src/plugins/unified_field_list/public/index.ts

```diff
@@ -43,6 +43,7 @@ export {
 } from './components/field_visualize_button';
 export { loadFieldStats } from './services/field_stats';
 export { loadFieldExisting } from './services/field_existing';
+export { getResolvedDateRange } from './utils/get_resolved_date_range';
```

Comment:
Do we use this function outside of unified field list? If not, does it need to be exported?